BackofficeBundle/Resources/public/less/smartadminPatches/nav.less

```diff
@@ -16,5 +16,38 @@ nav {
         }
       }
     }
+    > li > a b {
+      position: absolute !important;
+      right: 10px;
+      top: 10px;
+    }
   }
 }
+
+#left-panel nav ul li {
+  > a {
+    b.collapse-sign {
+      position: absolute;
+      top: 8px;
+      right: 10px;
+    }
+
+    span {
+      display: block;
+      margin-right: 15px;
+      overflow: hidden;
+      text-overflow: ellipsis;
+      white-space: nowrap;
+    }
+    span.menu-item-parent{
+      display: inline-block;
+      overflow: initial;
+    }
+  }
+}
+
+ol.breadcrumb {
```

Comment:
i think it's better to add this part in other file because this doesn't concern navigation